code/modules/clothing/head/tinfoilhat.dm

```diff
@@ -67,3 +71,24 @@
 	. = ..()
 	if(!warped)
 		warp_up()
+
+/obj/item/clothing/head/foilhat/proc/call_suicide(datum/source)
+	SIGNAL_HANDLER
+	INVOKE_ASYNC(src, /obj/item/clothing/head/foilhat.proc/suicide_act, source)	//SIGNAL_HANDLER doesn't like to wait, so i decided to move it to a proc in INVOKE_ASYNC
+	return OXYLOSS
+
+/obj/item/clothing/head/foilhat/suicide_act(mob/living/user)
+	user.visible_message(span_suicide("[user] gets a crazed look in [user.p_their()] eyes! [capitalize(user.p_they())] [user.p_have()] witnessed the truth, and try to commit suicide!"))
```

Comment:
c'mon, you know it ought to be "It looks like they're about to die of natural causes!" :)